src/System.Memory/tests/ReadOnlySpan/GcReporting.cs

```diff
@@ -0,0 +1,67 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.SpanTests
+{
+    public static class ReadOnlySpanGcReportingTests
+    {
+        [Fact]
+        public static void DelegateTest()
+        {
+            DelegateTest(100000, 10000);
```

Comment:
It takes a few seconds on a debug build. I have added `[OuterLoop]` to the test.